plugins/nv-filters/data/locale/en-US.ini

```diff
@@ -0,0 +1,16 @@
+NoiseSuppress.SuppressLevel="Suppression Level"
+NoiseSuppress.Intensity="Suppression Intensity"
+NoiseSuppress.Method="Method"
+NoiseSuppress.Method.Nvafx.Denoiser="NVIDIA Noise Removal"
+NoiseSuppress.Method.Nvafx.Dereverb="NVIDIA Room Echo Removal"
+NoiseSuppress.Method.Nvafx.DenoiserPlusDereverb="NVIDIA Noise Removal + Room Echo Removal"
+NoiseSuppress.Method.Nvafx.Deprecation="WARNING: Please upgrade both NVIDIA Video & Audio SDK. Your current version of Audio SDK is outdated."
```

Comment:
Since we're creating a new string key, maybe this should be NvFilters.Method.Nvafx.OutdatedSDK` instead, and we can use `NvFilters` or something else instead of `NoiseSuppress`?

Let me know if those changes are too involved. I just wanted to consider them.

Reply:
The string names come directly from obs-filters where the filter was NoisSuppress and the methods were: speex, rnnoise and nvafx with 3 variants. So yeah, we can remove the reference to NoiseSuppress; I've changed NoiseSuppress.Method.Nvafx to Nvafx.Method.
I've changed also to NvFilters.OutdatedSDK instead of NoiseSuppress.Method.Nvafx.Deprecation